charts/prefect-orion/Chart.yaml

```diff
@@ -1,21 +1,27 @@
 apiVersion: v2
-name: prefect-orion
+appVersion: latest
+dependencies:
+  - name: common
+    repository: https://charts.bitnami.com/bitnami
+    tags:
+      - bitnami-common
+    version: 2.0.2
```

Comment:
one thing to keep in mind about switching to a bitnami common base is that we'll have to keep this relatively updated to latest, since they started having to drop anything older than 6 months prior from their root manifest bc their cdn started timing out

https://github.com/bitnami/charts/issues/10539

Reply:
hm im not sure what you're suggesting @madkinsz

Reply:
good to know - I'll add an item to determine a good way to track upstream dependencies & releases